packages/shared/providers.yaml

```
@@ -4665,6 +4665,22 @@ typeform:
         base_url: https://api.typeform.com
     docs: https://docs.nango.dev/integrations/all/typeform
 
+typefully:
+    display_name: Typefully
+    categories:
+        - twitter
```

```suggestion
        - productivity
```

Also needs to be added to productivity.mdx